packages/build-utils/test/fixtures/22-pnpm/package.json

```
@@ -0,0 +1,15 @@
+{
+  "name": "22-pnpm",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "mkdir -p public && (printf \"pnpm version: \" && pnpm -v) > public/index.txt"
```

```suggestion
    "build": "mkdir -p public && printf \"pnpm version: \" && pnpm -v > public/index.txt"
```

The paranthesis aren't needed here